ement.el

```diff
@@ -222,6 +216,51 @@ call `pop-to-buffer'."
 
 ;;;; Functions
 
+(defun ement--read-username-and-server (prompt &optional initial-input history
```

Comment:
Do we really need all these extra arguments?  We aren't passing them, are we?

Reply:
• I miss a good input history already and was going to add it.
• In general I tend to do this when it's clear how interfaces map to each other.  All standard read- functions have the same interface. I realise some will object to this.  You do; OK.  Unless you're not convinced that your read- functions should share this interface, I'll retract the unused args.

Reply:
I don't think it's necessary for these prompts to have their own input histories, is it?  Most users will type in their MXID and password no more than once per session, and less often if they save the session.  I'd prefer to keep everything as simple as possible.